Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -2487,6 +2487,10 @@ public bool FillMissingEpisodeNumbersFromPath(Episode episode, bool forceRefresh
                 {
                     episode.ParentIndexNumber = season.IndexNumber;
                 }
+                else
+                {
+                    episode.ParentIndexNumber = 1;
```

Comment:
can you explain why?

Reply:
Perhaps this would  be better in tvdb parser. The problem seems to be the following:
Anime series don't generally have a season in their file name, however, tvdb needs a season to correctly get the metadata. Hence, a null season needs to be filled with something.

Reply:
at least add a comment explaining exactly this (or a `//FIXME` maybe)